(node:3412864) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'banner-session-40680',
  containerIp: '10.0.3.79',
  mitmPort: 40680,
  autoAccessFixed: 1
}
2024-11-06 10:27:56.303 - [Info] MITM Version: 2
2024-11-06 10:27:56.305 - [Info] Auto Access Enabled: true
2024-11-06 10:27:56.306 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-06 10:27:56.317 - [Info] SSH man-in-the-middle server for 10.0.3.79 listening on 10.0.3.1:40680
2024-11-06 10:28:53.657 - [Debug] [Connection] Attacker connected: 175.100.24.139 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-06 10:28:55.461 - [Debug] [Auth] Attacker 175.100.24.139 trying to authenticate with 'password'
2024-11-06 10:28:55.462 - [Debug] [Auto Access] Attacker: 175.100.24.139, Threshold: 1, Attempts: 1
2024-11-06 10:28:55.462 - [Debug] [Auto Access] Compromising the honeypot
2024-11-06 10:28:55.463 - [Debug] [Auto Access] Adding the following credentials: 'tv:123456'
2024-11-06 10:28:55.797 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-06 10:28:55.799 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-06 10:28:55.800 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.79
2024-11-06 10:28:55.890 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2024-11-06 10:28:57.145 - [Debug] [EXEC] Noninteractive mode attacker command: cd ~; chattr -ia .ssh; lockr -ia .ssh
2024-11-06 10:28:58.947 - [Debug] [Connection] Attacker connected: 175.100.24.139 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-06 10:29:00.606 - [Debug] [Auth] Attacker 175.100.24.139 trying to authenticate with 'password'
2024-11-06 10:29:00.607 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2024-11-06 10:29:00.608 - [Debug] [Auth] Attacker: 175.100.24.139 has so far made 1 attempts. Remaining: 5 attempts
2024-11-06 10:29:01.187 - [Debug] [Connection] Attacker closed the connection
2024-11-06 10:29:01.455 - [Debug] [Connection] Attacker connected: 175.100.24.139 | Client Identification: SSH-2.0-libssh_0.9.6
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
